src/components/main/flight/ReceiptWithFlightScreen.js

```diff
@@ -142,16 +142,16 @@ const styles = StyleSheet.create({
     flexDirection: "column",
     marginLeft: Math.round(moderateScale(105, 0.625)),
     marginRight: Math.round(moderateScale(20, 0.0625)),
-    marginBottom: Math.round(moderateScale(15, 0.0625)),
-    marginTop: Math.round(moderateScale(20, 0.0625)),
+    marginBottom: Math.round(moderateScale(30, 0.0625)),
+    marginTop: Math.round(moderateScale(10, 0.0625)),
```

Comment:
Why doesn't safe view work with android?

Reply:
> ![image](https://user-images.githubusercontent.com/5898380/87976049-51c9e800-ca81-11ea-8fbf-08eeb06a749f.png)
> 
> Please fix the margins so the top is balanced with the bottom. I suggest removing some of the spacing between the two values at the bottom (distance and carbon emissions)
> 
> Also, please align all items to the same left and right margin (shown by the red line) You can again fill up more of the empty space by making the airport abbreviations just a bit larger too (however you feel looks best)

Okay, I see what you are saying. I centered them, because the plane icon won't center between the two items using flex because if LHR is not centered there is trailing, "extra" white space from flex. So the Icon won't be in the middle.

Reply:
Bottom and Top margins are actually the same. Just that they are small: 5 and 5. Safe Area View works on other Android devices from my testing and adds a little nudge so I did not account for your device to be an outlier. I checked both with inspector and in the code. They are both 5. This may just be a singular case. I tested this flex model from the smallest Iphone to largest.

Reply:
Im going to do android tests and get back to you as to wether or not these margins should change and by how much to prevent it breaking the flex model. Margins have a way of bugging out how flex redistributes space. So I avoided using too much of them

Reply:
Make the LHR -> SFO a separate container from the actual airport name above it.

Oh, we stopped using safe area view since it was super janky and affects flex all around. Just leave ample margins and it should be fine. If you want to get super nitty gritty, you can also use grab the status bar height and use that as a constant.

Reply:
Bro, that results in more white space allocated for wrapping airport names and then you get the same imbalance except this time in white space. Flex doesn't behave the way you imagine in it will, I tried that approach first XD

Reply:
Also when did we DEP safe area view? I designed an entire page without knowing this convention and although its not that much of a set back. This entire pull request may need to be overhauled and scrapped for changes involving margins box sizes. Flex adjustments.

Reply:
In which files have we used safeViewArea?

Reply:
SafeAreaView* and before I started doing the flight screens, I am pretty sure every screen after Home screen had it. Sign Up, Receipt and Flight Info

Reply:
Oof, well I'm sorry. It worked for those pages so I guess you could honestly try making it work for this screen or scrap it. We started scrapping it but must not have removed all instances.

Reply:
Basically, you can keep using SafeAreaView, I've just found it easier to work without it. However, if it works, it works.

Reply:
### Flex Model Before:
-----------------------
<img width="487" alt="Screen Shot 2020-07-20 at 7 17 22 PM" src="https://user-images.githubusercontent.com/22000425/88010730-71393300-caca-11ea-8cdb-7450b06f1ebc.png">

### Flex Model After: 
---------------------
 ![Simulator Screen Shot - iPhone 11 - 2020-07-20 at 18 38 28](https://user-images.githubusercontent.com/22000425/88010725-6da5ac00-caca-11ea-9dab-c5473f07ba9e.png)

### Current Results: 
---------------------
<img width="487" alt="Screen Shot 2020-07-20 at 8 46 28 PM" src="https://user-images.githubusercontent.com/22000425/88010760-801fe580-caca-11ea-9a5c-6cbeada01f5e.png">

Reply:
tbh ^^ that's the closest it's gonna get with all of them centered like that. And way less wasted vertical white space ( which Is far more important).

Reply:
Let's just keep it how it is but lose the full airport name. so just LHR -> SFO with the current margins

Reply:
Can we augment that intro then to include airport names secondary to departure and arrival countries. States ETC?

Reply:
I actually found a fix for this. Im going to commit to a branch and you check it out.

Reply:
https://github.com/wang1kevin1/InstitutoTerra/compare/REF-7...REF-7-kwang